qiita_db/study.py

```diff
@@ -503,6 +515,21 @@ def info(self):
             # This is an optional column, but should not be considered part
             # of the info
             info.pop('study_id')
```

Comment:
The wording is bad, but I think it's supposed to mean it could be part of the main info, but we thought no. I'll reword, since really it's removed because it's redundant to the study id already stored in the object.